arch/arm/soc/st_stm32/stm32f7/Kconfig.series

```diff
@@ -0,0 +1,19 @@
+# Kconfig - ST Microelectronics STM32F7 MCU series
+#
+# Copyright (c) 2018 Yurii Hamann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_STM32F7X
+	bool "STM32F7x Series MCU"
+	select CPU_CORTEX_M7
+	select CPU_HAS_FPU
+	select SOC_FAMILY_STM32
+	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
+	select HAS_STM32CUBE
+	select CPU_HAS_MPU
```

Comment:
Sorry for late catch, but I'm not sure about activating MPU right now.
MPU for Cortex M7 has specific options which might affect performance.
Hence I prefer this is activated specific check/study rather than by default.

Reply:
OK. I will deactivate it.

Reply:
I have deactivated MPU and updated the commit.